lib/kube/proxy/forwarder.go

```diff
@@ -835,12 +843,18 @@ func (f *Forwarder) setupContext(ctx context.Context, authCtx auth.Context, req
 				),
 			)
 			defer span.End()
+
+			// Make sure we will only send signed PROXY headers to teleport kube service node, not real kube cluster
+			if forwarderType != ProxyService && endpoint.serverID == "" {
```

Comment:
since this is used in several places, can you move it near `clientSrc, clientDst := utils.ClientAddrFromContext(req.Context())`?

Reply:
But we're using endpoint parameter scoped to this dial function?